functions/test/endpoints/PointLog-Tests/Handle.test.ts

```diff
@@ -2,20 +2,16 @@ import * as factory from '../../HTTPRequestFactory'
 import * as firebase from "@firebase/testing"
 import * as IntegrationMockFactory from '../IntegrationMockFactory'
 import * as request from 'supertest'
-import {FirestoreDataFactory, POINT_LOG_DEFAULTS} from '../FirestoreDataFactory'
+import {FirestoreDataFactory} from '../FirestoreDataFactory'
+import { POINT_LOG_DEFAULTS } from '../../OptionDeclarations'
 
 let point_log_func
 let db: firebase.firestore.Firestore
 let approved_log
 let unhandled_log
 let rejected_log
-let point_value
 let point_description
-let house_doc_ref
-let house_points
-let user_doc_ref
-let user_points
-let sem_points
+
```

Comment:
These global variables have been removed and the new CompetitionPointStatus check will be used